pom.xml

```diff
@@ -71,6 +71,11 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>2.13.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>6.1.5.Final</version>
```

Comment:
Did you try to use oldest version? e.g. `8.0.0.Final`

Reply:
I tried to use 8.0.1.Final. But i got problems with this. 8+ version demands jakarta.validation instead of javax.validation and validation didn't work with this. That's why I decided to take the most popular one on mvnrepository